index.html

```diff
@@ -6,7 +6,7 @@
     <!-- link to CSS code -->
     <link rel="stylesheet" href="tictactoe.css">
     <!-- link to JavaScript code -->
-    <script src="scripts.js" defer></script>
+    <script src="scripts.js"></script>
```

Comment:
if you are setting the `<scripts>` tag in the `<head>`, you want to keep the `defer `attribute